tasks/agent6.py

```diff
@@ -0,0 +1,33 @@
+from invoke import task
+
+from tasks.libs.common.agent6 import agent_context, enter_agent6_context, prepare
+
+
+@task
+def init(ctx):
+    """Will prepare the agent 6 context (git clone / pull of the agent 6 branch)."""
+
+    prepare(ctx)
+
+
+@task
+def run(ctx, command):
+    """Runs a command in the agent 6 environment."""
+
+    with agent_context(ctx, 6):
+        ctx.run(command)
+
+
+@task
+def invoke(ctx):
+    """Enters the agent 6 environment in order to invoke tasks in this context.
+
+    Note:
+        This task should be avoided when a --version, --major-version or --agent-version argument is available in the task.
```

Comment:
I suppose we should update the [release coordinator guide](https://datadoghq.atlassian.net/wiki/spaces/agent/pages/3649801154/Release+Coordinator+guide) with what we must call in the agent6 context